courses/CEIR12.md

```diff
@@ -0,0 +1,104 @@
+---
+code: CEIR12
+title: Engineering Graphics
+
+specifics:
+  - branch: CE
+    semester: 1
```

Comment:
I wonder if this might be `2` because the course itself requires a prereq that was taught in sem 1.

Reply:
Checked [the Civil Engineering scheme](https://drive.google.com/file/d/16GZfj3pzUaiKMx-frey_K2dB2Oy6xe0d/view), seems this is in the first semester alongside CEIR11 and its prerequisite.

Reply:
The scheme is poorly written. If you check sem 2, the same course is mentioned there as well. It's ambiguous with MEIR11 IIRC.